pysat/instruments/methods/nasa_cdaweb.py

```diff
@@ -74,7 +74,7 @@ def list_files(tag=None, sat_id=None, data_path=None, format_str=None,
             try:
                 format_str = supported_tags[sat_id][tag]
             except KeyError:
-                raise ValueError('Unknown tag')
+                raise ValueError('Unknown tag: ' + tag)
```

Comment:
I suppose a bad sat_id could also cause a KeyError as well. How about a unknown sat_id and tag combo error statement?

Reply:
Will update

Reply:
fixed